Polish/przygoda/wola/eliksir/eliksir.md

```diff
@@ -0,0 +1,3 @@
+Wypiłeś tyle eliksiru woli, że spojrzeniem zmiotłeś czasoprzestrzeń. Wśród kompletnej pustki dostrzegasz tunel, który przenosi cię w czasie. Jesteś w przyszłości w której ludzie zapomnieli o Johnie Travolcie w "Gorączce Sobotniej Nocy". 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.